README.md

```diff
@@ -4,11 +4,32 @@
 
 This repository contains [Helm](https://helm.sh/) charts related to [Lagoon](https://github.com/amazeeio/lagoon/).
 
+## Table of Contents
```

Comment:
Github auto-generates a TOC (although it's a bit hidden). This one being in-line is more discoverable, but does it duplicate the existing functionality?
https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Reply:
@tobybellwood / @AlannaBurke any thoughts on this?